modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_set.scss

```diff
@@ -50,78 +50,145 @@ body {
 
 // Buttons
 
+.btn-danger {
+	@include custom-properties-button(
+		var(--danger),
+		var(--danger),
+		var(--white)
+	);
+}
+
+.btn-dark {
+	@include custom-properties-button(var(--dark), var(--dark), var(--white));
+}
+
+.btn-info {
+	@include custom-properties-button(var(--info), var(--info), var(--white));
+}
+
+.btn-light {
+	@include custom-properties-button(var(--light), var(--light), var(--dark));
+}
+
 .btn-link:not(.dropdown-item) {
-	@include button-cp(
+	@include custom-properties-button(
 		var(--transparent),
```

Comment:
Not sure why we need a `--transparent` variable. I don't see anyone wanting to change this one.

Reply:
I don't know why we have that variable 😁 the only usage that come to my mind is to make outlined buttons with a base color